apps/web/lib/team/[slug]/getServerSideProps.tsx

```diff
@@ -78,26 +79,22 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     return { notFound: true } as const;
   }
 
-  if (!team || (team.parent && !team.parent.slug)) {
+  if (!team) {
```

Comment:
Separate a regular team and a sub-team handling. 
Because regular team would still remain unpublished always but sub team would depend on if it was redirected from nonOrgLink or not.